code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -2155,20 +2155,24 @@
 	name = "Dangles"
 	desc = "Once you pop, you'll wish you stopped."
 	icon_state = "dangles"
-	trash = /obj/item/trash/danitos
+	trash = /obj/item/trash/dangles
 	filling_color = "#FF9933"
 	base_crumb_chance = 30
+	var/popped
+
+/obj/item/weapon/reagent_containers/food/snacks/dangles/can_consume(mob/user)
+	return popped
 
 /obj/item/weapon/reagent_containers/food/snacks/dangles/attack_self(var/mob/user)
-	if(!is_open_container())
+	if(!popped)
```

Comment:
is `popped` ever initialised?
you set it true in pop_open but that happens after this check

Reply:
L#2161, defaults to `null` which afaik resolves as `FALSE`